src/onegov/org/layout.py

```diff
@@ -1719,19 +1720,28 @@ def breadcrumbs(self):
 
     @cached_property
     def editbar_links(self):
-        if self.request.is_manager:
-            return [
-                Link(
+        def links():
+            if self.request.is_admin and isinstance(self.app, WinterthurApp):
+                yield Link(
+                    text=_("Configure"),
+                    url=self.request.link(self.model, '+edit'),
+                    attrs={'class': 'edit-link'}
+                )
```

Comment:
Why are you restricting this to `WintherthurApp`? It doesn't make any sense to me.

If you really wanted to do that, then this is the wrong place anyways, you would need to overwrite the layout in `onegov.winthethur` and also override all the views that use it.

Reply:
As mentioned above, I think we should use either tags or filters. How should one decide? By setting the filter config? Or would you introduce a setting?

Reply:
If you go the settings route, the setting should probably be `tags`, `filters` and `tags_and_filters` with the default being `tags`.

Reply:
The filter specific configuration fields can then only be displayed when either of the former two values has been selected.

Reply:
Limitation to winterthur removed, filters are generally available and can be configured in organisation settings